src/Gui/Dialogs/DlgParameterImp.cpp

```diff
@@ -986,7 +986,7 @@ void ParameterValue::onCreateBoolItem()
     QStringList list;
     list << QStringLiteral("true") << QStringLiteral("false");
     QString val = QInputDialog::getItem(this,
-                                        QObject::tr("New boolean item"),
+                                        QObject::tr("New Boolean Item"),
                                         QObject::tr("Choose an item:"),

```

Comment:
```suggestion
                                        QObject::tr("Choose an item"),

```

would you choose "an item" if you create a new boolean item? Or just call it: "Boolean Value"